framework/src/task_manager/task_manager_core.c

```diff
@@ -556,59 +562,154 @@ int taskmgr_get_handle_by_pid(int pid)
 	return TM_UNREGISTERED_APP;
 }
 
+static int taskmgr_get_tm_broad_msg(int msg)
+{
+	int tm_broadcast_msg_idx;
+	
+	for (tm_broadcast_msg_idx = 0; tm_broadcast_msg_idx < CONFIG_TASK_MANAGER_MAX_TASKS; tm_broadcast_msg_idx++) {
+		if (tm_broadcast_msg[tm_broadcast_msg_idx].input_msg == msg) {
+			return tm_broadcast_msg[tm_broadcast_msg_idx].tm_broad_msg;
+		}
+	}
+	return TM_INVALID_BROADCAST_MSG;
+}
+
+static tm_broadcast_t *taskmgr_search_broadcast_info(int msg, int handle)
+{
+	int tm_broad_msg;
+	sq_entry_t curr;
+
+	tm_broad_msg = taskmgr_get_tm_broad_msg(msg);
```

Comment:
checking the return of taskmgr_get_tm_broad_msg is not needed?

Reply:
Done